MonoGame.Framework/BoundingBox.cs

```diff
@@ -40,9 +50,17 @@ public BoundingBox(Vector3 min, Vector3 max)
 
         #region Public Methods
 
+
+        /// <summary>
+        /// Determines the <see cref="ContainmentType"/> between this <see cref="BoundingBox"/> and another.
+        /// </summary>
+        /// <param name="box">The <see cref="BoundingBox"/> to compare to.</param>
+        /// <returns>The <see cref="ContainmentType"/> relationship between the two objects.</returns>
         public ContainmentType Contains(BoundingBox box)
         {
-            //test if all corner is in the same side of a face by just checking min and max
+            // If, in any dimension: the other box's MAX is less than this box's MIN, OR
+            //                       the other box's MIN is is more than this box's MAX
+            //                    then the two must be disjoint
             if (box.Max.X < Min.X

```

Comment:
I suggest avoiding adding these sorts of comments within functions unless you are the author of the code in question.

It is hard for us to decide if your comment is valid or not and we don't have the time to investigate it.